pkg/apis/gcpprovider/v1beta1/gcpmachineproviderconfig_types.go

```diff
@@ -104,3 +108,11 @@ type GCPKMSKeyReference struct {
 	// Location is the GCP location in which the Key Ring exists.
 	Location string `json:"location"`
 }
+
+// GCPAcceleratorConfig describes type and count of accelerator cards attached to the instance on GCP.
+type GCPAcceleratorConfig struct {
+	// AcceleratorCount is number of AcceleratorType accelerators (GPUs) to be attached to an instance
+	AcceleratorCount int64 `json:"acceleratorCount,omitempty"`
+	// AcceleratorType is the type of accelerator (GPU) to be attached to an instance
```

Comment:
Makes sense since it is for the `oc describe`

Btw supported accelerators are listed in a structure https://github.com/openshift/cluster-api-provider-gcp/pull/172/files#diff-359c93fbc08f569c3abf79881f0b426e681c46a5e76765d05e921f16fdb3bf99R44